lib/gems/pending/util/extensions/miq-array.rb

```diff
@@ -1,14 +1,17 @@
 require 'more_core_extensions/core_ext/array'
 require 'active_support/core_ext/array/wrap'
+require 'active_support/deprecation'
 
 class Object #:nodoc:
   def to_miq_a
+    ActiveSupport::Deprecation.warn("Object#to_miq_a should be replaced by Array.wrap", caller.drop(1))
     Array.wrap(self)
   end
 end
 
 class String
   def to_miq_a
+    ActiveSupport::Deprecation.warn("String#to_miq_a should be replaced by #lines.to_a", caller.drop(1))
```

Comment:
Maybe it should include the String class: `String#lines.to_a` to be consistent with the other messages here.